src/actions/roles.js

```diff
@@ -23,30 +23,6 @@ export function getRolesByNames(rolesNames: Array<string>) {
     });
 }
 
-export function getRoleByName(roleName: string) {
-    return bindClientFunc({
-        clientFunc: Client4.getRoleByName,
-        onRequest: RoleTypes.ROLE_BY_NAME_REQUEST,
-        onSuccess: [RoleTypes.RECEIVED_ROLE, RoleTypes.ROLE_BY_NAME_SUCCESS],
-        onFailure: RoleTypes.ROLE_BY_NAME_FAILURE,
-        params: [
-            roleName,
-        ],
-    });
-}
-
-export function getRole(roleId: string) {
```

Comment:
Same as the above comment.

Reply:
Can you add the action here back? As i was saying it is better to have the actions in the file. Just removing the `onRequest`, `onSuccess` and `onFailure` is good enough